app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -64,9 +64,10 @@ def forgot_password_email_confirmed(success:, failure_reason: nil)
 
     # The user has exceeded the rate limit for password reset emails
     # @param [String] email The user's email address
-    def forgot_password_email_rate_limited(email:)
+    def forgot_password_email_rate_limited(email:, success:)
       track_event(
         :forgot_password_email_rate_limited,
+        success: success,
```

Comment:
We may not want to merge this, looking at `app/services/irs_attempts_api/tracker_events.rb`, the following rate limited events presently do not contain `success: true`